alcs-api/src/application/application.service.ts

```diff
@@ -177,4 +188,15 @@ export class ApplicationService {
       paused: appPausedMap.get(app.uuid) || false,
     }));
   }
+
+  searchApplicationsByFileNumber(fileNumber: string) {
```

Comment:
Are you leaving this is in in anticipation of adding a search? (Looks like its unused)

Reply:
it is used for autocomplete the list of applications once user starts inputting file number on recon creation form